calendar_resource/models/__init__.py

```diff
@@ -2,5 +2,8 @@
 # Copyright 2014 Therp BV
```

Comment:
This is the only Therp I see too. Perhaps someone just copied an init file and forgot to change copyright?